docs/docs/guides/local-development.md

```diff
@@ -94,6 +94,22 @@ pnpm run test:watch    # Run tests in watch mode
 pnpm run lint          # Lint code
 ```
 
+### Direct Client Chat UI
+
+```
+# Open a terminal and Start with specific character
+pnpm run dev --characters="characters/my-character.json"
+```
+```
+# Open a 2nd terminal and go to the client directory
```

Comment:
I believe we don't have to go to the `client` directory to run the script as we already specify `--dir client` in `start:client` script.